sdk/identity/Azure.Identity/api/Azure.Identity.netstandard2.0.cs

```diff
@@ -245,6 +245,7 @@ public partial class TokenCredentialOptions : Azure.Core.ClientOptions
     {
         public TokenCredentialOptions() { }
         public System.Uri AuthorityHost { get { throw null; } set { } }
+        public bool PreferClientConfiguredTenantId { get { throw null; } set { } }
```

Comment:
I think the way this switch behaves will be problemattic in the ARM case where we have to fetch tokens for multiple tenants up front. It seems like it would give us the same token each time. Also, I think the name is confusing because the end user never configures the tenant id through the client. Perhaps this should be something more like `AllowMutltiTenantAuthentication`?